src/main/java/com/locationinformationmicroservice/facade/LocationInformationFacade.java

```diff
@@ -6,25 +6,126 @@
 import com.locationinformationmicroservice.service.ImageService;
 import com.locationinformationmicroservice.service.WikiService;
 
+import com.locationinformationmicroservice.model.Image;
+import com.locationinformationmicroservice.ImageRepository;
+
+import java.util.Optional;
+import java.util.ArrayList;
+import java.util.List;
 
 public class LocationInformationFacade {
-  public static HashMap<String, Object> getDetails (String lat, String lon, String image_api_key, String geo_api_key) {
+  public static HashMap<String, Object> getDetails (String lat, String lon, String image_api_key, String geo_api_key, ImageRepository imageRepository) {
```

Comment:
We will need to look into how not to have the `ImageRespository` required as a passed in parameter.